.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test rosbag2
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
shouldn't this point to `tmoulard/gh-action-ci-build-failure`

Reply:
This just describes what triggers a build: a push on master (for the post-merge check) or a PR. For PRs, whatever branch it is merged on will be used.